Objects/odictobject.c

```diff
@@ -1776,8 +1788,10 @@ odictiter_iternext(odictiterobject *di)
 PyDoc_STRVAR(reduce_doc, "Return state information for pickling");
 
 static PyObject *
-odictiter_reduce(odictiterobject *di, PyObject *Py_UNUSED(ignored))
+odictiter_reduce(PyObject *op, PyObject *Py_UNUSED(ignored))
 {
+    odictiterobject *di = (odictiterobject*)op;
+
```

Comment:
```suggestion
```

Reply:
IMO this empty line helps readability.

Reply:
Oh, I thought it was a typo due to some C/C. My bad!